homeassistant/components/unifi/controller.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, hass, config_entry, api):
 
         api.callback = self.async_unifi_signalling_callback
 
-        self.available = True
+        self.available: bool = True
```

Comment:
This should be inferred. It isn't because `__init__` isn't typed.
I think it's better to first type `UniFiController.__init__` method, and after that is done then you enable strict typing to switch.

Reply:
This blows up to way too many mypy errors so I will just park this PR for the moment as I will have to type so much more

Reply:
Finally got to a point where this wouldn't be an unsurmountable amount of work https://github.com/home-assistant/core/pull/90278 🎉